python/ray/train/callbacks/logging.py

```diff
@@ -174,6 +175,102 @@ def _validate_worker_to_log(self, worker_to_log) -> int:
         return worker_to_log
 
 
+class MLflowLoggerCallback(TrainingSingleWorkerLoggingCallback):
+    """MLflow Logger to automatically log Train results and config to MLflow.
+
+    MLflow (https://mlflow.org) Tracking is an open source library for
+    recording and querying experiments. This Ray Train callback
+    sends information (config parameters, training results & metrics,
+    and artifacts) to MLflow for automatic experiment tracking.
+
+    Args:
+        tracking_uri (Optional[str]): The tracking URI for where to manage
+            experiments and runs. This can either be a local file path or a
+            remote server. This arg gets passed directly to mlflow
+            initialization.
+        registry_uri (Optional[str]): The registry URI that gets passed
+            directly to mlflow initialization.
+        experiment_id (Optional[str]): The experiment id of an already
+            existing experiment. If not
+            passed in, experiment_name will be used.
+        experiment_name (Optional[str]): The experiment name to use for this
+            Train run.
+            If the experiment with the name already exists with MLflow,
+            it will be used. If not, a new experiment will be created with
+            this name.
```

Comment:
Could you elaborate more on what you'd like to see here?

There is information in the description for the arguments on the behavior if None is passed in.

Reply:
Ah, so I'm not clear from this doc what the behavior is if `tracking_uri` or `registry_uri` is `None`. Taking another look at the implementation and docs I now see that it's mentioned in `logdir` doc - it would be helpful to include this information in each of these parameter docs as well.

I also can't tell from this doc at all what happens if `experiment_name` is `None` (and the corresponding environment variable is not set). From the error message, it seems like at least `experiment_name` or `experiment_id` must be set.

Reply:
Got it, thanks for the explanation! Added it to the docstring.